Packs/Lastline/ReleaseNotes/1_0_12.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Lastline v2
+- Fixed an issue where the ***lastline-upload-file*** command failed due to an IndexError exception.
```

Comment:
In what cases would the `IndexError` happen? I'm worried the pythonic error might not be helpful for customers.
May I suggest:
Fixed an issue where the ***lastline-upload-file*** command failed for certain inputs.